modules/commsInterface/commsInterface.py

```diff
@@ -0,0 +1,74 @@
+import usb.core
+import usb.util
+import serial
+
+class CommsInterface:
+    def __init__(self, type: bool):
+        """
+        Initializes parameters for USB or UART
+
+        Parameters
+        ----------
+        type: bool
+            Boolean used to specify selection of USB or UART;
+            0|False for USB, 1|True for UART
+        """
+        self.uart_or_usb = type
+    
+    def create_end_point_FC(self):
+        """
+        Issues a read command in either serial (UART) or pyusb (USB).
+        EndpointId can be the specific serial port to use or the usb endpoint to be used. 
+        """
+        if self.uart_or_usb == 0:
```

Comment:
Honestly this isn't quite how I was envisioning you'd do this but it's kinda cool so we'll keep it. As long as it works.

Reply:
I could probably be ultra nitpicky and say use an enum, but I don't care enough cause we're using UART lol